src/main/template/float/com/flowpowered/math/vector/VectorN_E_.template

```diff
@@ -476,5 +487,10 @@ public class VectorN#E# implements Vector#E#, Comparable<VectorN#E#>, Serializab
         public void set(int comp, #e# val) {
             throw new UnsupportedOperationException("You may not alter this vector");
         }
+
+        @Override
+        public void setZero() {
```

Comment:
How about also adding this and setIdentity to MatrixN (if it doesn't exit already)?